cachi2/core/package_managers/rpm/main.py

```diff
@@ -378,3 +380,37 @@ def _generate_repofiles(
 
             with open(repo_file_path, "w") as f:
                 repofile.write(f)
+
+def _get_ssl_context():
+    client_cert = getenv("C2_CLIENT_CERT")
+    client_key = getenv("C2_CLIENT_KEY")
+    ca_bundle = getenv("C2_CA_BUNDLE")
+    
+    ssl_ctx = ssl.create_default_context()
+
+    if client_cert is None or client_key is None:
+        log.info(f"No client certificates will be used.")
+    elif not path.isfile(path=client_cert) or not path.isfile(path=client_key) :
+        raise(FileNotFoundError)
```

Comment:
This is not a good idea because in case of an error this crashes with a traceback. You need a `Cachi2Error` subclass error here for the code to end with a user friendly error, you'll most likely be interested in the `FetchError`.